docs/index.html

```diff
@@ -0,0 +1,13 @@
+<HTML>
+<HEAD>
+<TITLE>Litmus Helm Chart</TITLE>
+<META HTTP-EQUIV="refresh" CONTENT="2;URL=./litmus-operator-latest.yaml">
+</HEAD>
+
+<BODY>
+
+Your browser should automatically take you there in 10 seconds. If it doesn't plase go to https://litmuschaos.github.io/chaos-charts/litmus-operator-latest.yaml
```

Comment:
I guess we can setup gh-pages to to https://litmuschaos.github.io/litmus/litmus-operator-latest.yaml

Reply:
What do you mean?

Reply:
i.e., the older operator manifest was being pulled from this link: //litmuschaos.github.io/pages/litmus-operator-latest.yaml (if you look at the docs etc.,). Now that we are placing this into litmus repo, we should ideally change this download link to https://litmuschaos.github.io/litmus/litmus-operator-latest.yaml (and not chaos-charts) 

This can be setup by creating a gh-pages branch on this repo (like we did w/ pages and litmus-helm repo).

Reply:
Refer: https://docs.github.com/en/enterprise/2.14/user/articles/configuring-a-publishing-source-for-github-pages

Reply:
Thanks for references and explanation, will take a look!